### PR TITLE
Normalize error message

### PR DESCRIPTION
**Pull request by jviotti**
Signed-off-by: Juan Cruz Viotti <jviotti@openmailbox.org>

### CODE REVIEW

File: lib/elevator.js [hunk 1]
```diff
@@ -72,10 +72,17 @@ exports.execute = function(command, options, callback) {
...
+        return callback.call(this, utils.ElevateCancelledError);
+      }
+
+      var elevationError = new Error('Couldn\'t elevate, exit code ' + error.code);
+      elevationError.description = error.message;
+      return callback(this, elevationError);
     }
 
+
     return callback.apply(this, arguments);
```
Comment (lurch) [inline lib/elevator.js:86]: Just out of curiosity, what do the different syntaxes of:
```js
return callback.call(this, utils.ElevateCancelledError);
return callback(this, elevationError);
return callback.apply(this, arguments);
```
all mean?

Comment (jviotti) [reply lib/elevator.js:86]: Damn,

```js
return callback(this, elevationError);
```

should have been:

```js
return callback.call(this, elevationError);
```

I'll send a fix.

The `.call` and `.apply` function are able to invoke a function with a custom context. Their first argument is the context, which is `this` in this case, so we're sure the context of this function is preserved.

The second argument represents the arguments with which the function should be called. Both `.call` and `.apply` do the same thing, but they differ on the argument syntax they accept. `.call` works like this: `function.call(context, argument1, argument2, argument3, ...)`, while `.apply` accepts an array of arguments: `function.apply(this, [ argument1, argument2, argument3, ...])`

Comment (jviotti) [reply lib/elevator.js:86]: See https://github.com/resin-io-modules/elevator/pull/12

Comment (lurch) [reply lib/elevator.js:86]: > Damn,

Wow, good thing I asked then :wink:

**Pull request merged**